openid-4-verifiable-presentations-1_0.md

```diff
@@ -215,6 +215,53 @@ Figure: Cross Device Flow
 
 (3) The Wallet prepares the Verifiable Presentation(s) of the Verifiable Credential(s) that the End-User has consented to. It then sends to the Verifier an Authorization Response where the Verifiable Presentation(s) are contained in the `vp_token` parameter.
 
+## Advanced Flow with Wallet Capabilities Discovery {#adv_flow}
```

Comment:
```suggestion
## Cross Device Flow with Wallet Capabilities Discovery {#adv_flow}
```
Not sure I am a big fan of "advance flow" term - feels too abstract.

Reply:
Yes, I'd tend to not use comparative terms like advanced or High or anything that might produce a relation or reference to something lower or feature-less

Reply:
I think the way this section is written right now does not exactly convey why this is an "advanced" flow. maybe i am misunderstanding something but I would expect more focus on the new ability for the wallet to communicate its capabilities/endpoint/attestation

Reply:
Advanced is the title of the PR, now it seems it will be a discovery (pre-step). Changed title and first sentence.